tests/virtiofs/datavolume.go

```diff
@@ -357,7 +357,7 @@ var _ = Describe("[sig-storage] virtiofs", decorators.SigStorage, func() {
 			Expect(err).ToNot(HaveOccurred())
 			libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 		},
-			Entry("unprivileged virtiofsd", testsuite.NamespaceTestDefault),
+			Entry("unprivileged virtiofsd", decorators.Conformance, testsuite.NamespaceTestDefault),
```

Comment:
I am not sure you can add this to `Conformance`: isn't this table playing with cluster-wide `kvconfig.DisableFeatureGate(virtconfig.VirtIOFSGate)` ?